ui/src/pages/createAccount.tsx

```
@@ -103,27 +100,24 @@ export default function CreateAccountPage() {
-                        stroke-linecap="round"
-                        stroke-linejoin="round"
-                        stroke-width="2"
-                        d="M5 13l4 4L19 7"
-                      />
-                    </svg>
-                    <span>{desc}</span>
-                  </li>
-                )
-              )}
+              roleDescriptions[role() as Role]?.map((desc: string, index: number) => (
+                <li class="flex items-center space-x-3 text-white p-2 rounded-md transition-colors">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)